<launch>
	<node name="teleop_twist_keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" output="screen">
		<remap from="cmd_vel" to="/bsc_jackal/key_vel"/>
	</node>
</launch>